### ISSUE TITLE
Data binding only works one way

### ISSUE THREAD

**Issue by Haxxer** [maxeckel/alpine-editor#2]
Hey,

We've been running into some problems getting data out of the editor. We can put data into it easy enough, but any references to `content` in any other model doesn't seem to update.

A simple test would be to use the test file you provide, and adding an additional div with `x-html="content"` on it, and when typing into the editor the content of the element won't update.

Lemme know if you need anything else.

Cheers.

**Comment by dashpilot** [comment]
I ran into the same issue. As a quick fix you could listen for the @input event of the editor, and update the model accordingly, for example:

```
<div data-type="editor" class="p-2" x-on:input="item.body = document.querySelectorAll('.ProseMirror')[0].innerHTML"></div>
```

**Comment by maxeckel** [comment]
Hey Folks,

this Bug is fixed now in v0.3.2. Thank you both for reporting this!

Cheers!
